app/src/test/java/com/example/habitsmasher/HabitSmasherTestSuite.java

```diff
@@ -0,0 +1,10 @@
+package com.example.habitsmasher;
+
+import com.example.habitsmasher.ui.dashboard.HabitListTest;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({HabitListTest.class})
```

Comment:
This is where we would add more test classes as we create them. This test suite is not that useful right now, as we only have one single test class, but as we create more, you would only have to run this class to run all tests in the project.